setup.py

```diff
@@ -33,7 +33,8 @@
     },
     keywords=['gis'],
     install_requires=[
-        # 'package==1.0.*'
+        'agrc-supervisor==2.0.*',
+        'pysftp==0.2.9',
```

Comment:
**question**: should pysftp be a dependency of palletjack? Is pallet jack a local module then?

Reply:
Good catch, leftover from before splitting out palletjack.